monitoring/messagesigning/config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+class Config:
+    BASE_PATH = os.path.dirname(__file__)
+    PRIVATE_KEY_PATH = "{}/keys/mock_priv.pem".format(BASE_PATH)
```

Comment:
Those configuration input should be moved to `mock_uss/config.py`. Again, depending on the design document requested in https://github.com/interuss/dss/pull/893, we can agree on the name of those configuration flags.